exercise1.js

```diff
@@ -0,0 +1,18 @@
+
+function calculateArrayAverage(numbers) {
+    // var sum = 0;
+    // var count = 0;
+    // for (var i = 0; i < numbers.length; i++) {
+    //   sum += numbers[i];
+    //   count++;
+    // }
+    // var average = sum / count;
+    // return average;
+
+    return numbers.reduce((a, b) => a + b , 0) / numbers.length;
```

Comment:
This is good! For the sake of clarity, I can suggest you rename a and b to reflect the meaning of the value they hold. 
For example:
```js
return numbers.reduce((acc,value) => acc + value , 0) / numbers.length
```